src/test/java/command/CheckCommandTest.java

```diff
@@ -0,0 +1,56 @@
+package command;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+import org.junit.jupiter.api.Test;
+
+import task.Deadline;
+import task.Event;
+import task.TaskList;
+
+public class CheckCommandTest {
+    private static final LocalDate DATE_1 = LocalDate.parse("01/01/2025",
+            DateTimeFormatter.ofPattern("dd/MM/yyyy"));
+    private static final LocalDate DATE_2 = LocalDate.parse("02/01/2025",
+            DateTimeFormatter.ofPattern("dd/MM/yyyy"));
+    private static final LocalDate DATE_3 = LocalDate.parse("03/01/2025",
+            DateTimeFormatter.ofPattern("dd/MM/yyyy"));
```

Comment:
Perhaps the numbers in the variable names can be changed to something that has a clearer explanation. The Coding quality guide indicates not use numbers in our variable names but I am also not too sure if it should apply in this case.
```suggestion
    private static final LocalDate DATE_FIRST_JAN = LocalDate.parse("01/01/2025",
            DateTimeFormatter.ofPattern("dd/MM/yyyy"));
    private static final LocalDate DATE_SECOND_JAN = LocalDate.parse("02/01/2025",
            DateTimeFormatter.ofPattern("dd/MM/yyyy"));
    private static final LocalDate DATE_THIRD_JAN = LocalDate.parse("03/01/2025",
            DateTimeFormatter.ofPattern("dd/MM/yyyy"));
```